custom_components/tesla_custom/manifest.json

```diff
@@ -4,7 +4,7 @@
   "config_flow": true,
   "documentation": "https://github.com/alandtse/tesla",
   "issue_tracker": "https://github.com/alandtse/tesla/issues",
-  "requirements": ["teslajsonpy==1.4.1"],
+  "requirements": ["teslajsonpy==1.5.0"],
```

Comment:
We'll also need to update pyproject.toml and run a poetry update.

Reply:
Forgot to push the new pyproject.toml

How do I run poetry update in github?

Reply:
It needs to be done at the shell.